src/libos/src/fs/disk_file.rs

```diff
@@ -131,6 +134,25 @@ impl DiskFile {
     pub fn ioctl(&self, _cmd: &mut dyn IoctlCmd) -> Result<()> {
         return_errno!(EINVAL, "this file does not support ioctl");
     }
+
+    pub fn metadata(&self) -> Metadata {
+        Metadata {
+            dev: 0,
+            inode: 0x1000,
```

Comment:
I think the value zero or a very large number would make a safer choice. The value `0x1000` could coincide with a valid inode number.

Reply:
Many apps/libs use zero as an invalid INode number, so the id should not be zero.